services/wallet/reader.go

```diff
@@ -89,13 +94,22 @@ func getTokenAddresses(tokens []*token.Token) []common.Address {
 	return res
 }
 
-func (r *Reader) Start(ctx context.Context, chainIDs []uint64) error {
-	accounts, err := r.s.accountsDB.GetAccounts()
-	if err != nil {
-		return err
-	}
-
-	return r.s.transferController.CheckRecentHistory(chainIDs, getAddresses(accounts))
+func (r *Reader) Start() error {
+	ticker := time.NewTicker(10 * time.Minute)
+	ctx := context.Background()
+	go func() {
+		for {
+			select {
+			case <-ctx.Done():
```

Comment:
Is ctx.Done() called when `Logout()` is called? if not, this goroutine will run forever